src/main/java/seedu/notus/storage/StorageManager.java

```diff
@@ -280,4 +310,62 @@ public boolean noteExists(Note note, boolean isArchive) {
         }
         return true;
     }
+
+    /* Timetable Saving and Loading */
+
+    /**
+     * Saves all the Events in the Timetable to the storage file.
+     *
+     * @param timetable The Timetable containing all the events to be saved.
+     */
+    public void saveTimetable(Timetable timetable) throws IOException {
+        String path = FOLDER_DIR + TIMETABLE_FILE_PATH;
+
+        //clear file
+        FileWriter fwClear = new FileWriter(path, false);
+        fwClear.write("");
+        fwClear.close();
+
+        //rewrite information to the file
+        FileWriter fwAppend = new FileWriter(path, true);
+
+        ArrayList<Event> nonRecurringEvents = timetable.getAllNonRecurringEvents();
+        String eventDetails;
+
+        for (Event event: nonRecurringEvents) {
+            eventDetails = getEventDetailsSaveFormat(event);
+            fwAppend.write(eventDetails);
+        }
+
+        ArrayList<RecurringEvent> recurringEvents = timetable.getAllRecurringEventsArray();
+
+        for (RecurringEvent event: recurringEvents) {
+            eventDetails = getEventDetailsSaveFormat(event);
+            eventDetails += PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_RECURRING
+                        + " " + event.getRecurrenceType() + " "
+                        + PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_STOP_RECURRING
+                        + " " + event.getEndRecurrenceDate() + " "
+                        + LS;
+            fwAppend.write(eventDetails);
+        }
+        fwAppend.close();
+    }
```

Comment:
So it's basically to convert the event back to it's input line and stored right? E.g. add-e /t Tutorial /timing 2020-10-29 13:00 /repeat weekly /reminder 1-day. Looks great!